# Created at 2022-05-31T14:34:40.039
org.testng.TestNGException: 
Two tests in the same suite [sanity_generalIns] cannot have the same name: add general ins to patient
	at org.testng.xml.internal.XmlSuiteUtils.ensureNoDuplicateTestsArePresent(XmlSuiteUtils.java:67)
	at org.testng.xml.internal.XmlSuiteUtils.validateIfSuitesContainDuplicateTests(XmlSuiteUtils.java:28)
	at org.testng.TestNG.sanityCheck(TestNG.java:968)
	at org.testng.TestNG.run(TestNG.java:993)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)


